CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Change Log
+
+Resources for generating a changelog:
+
+[skywinder/Github-Changelog-Generator](https://github.com/skywinder/Github-Changelog-Generator) - generates a full changelog that overwrites the existing CHANGELOG.md.
+
+[hzalaz/wt-oss-milestone-changelog](https://github.com/hzalaz/wt-oss-milestone-changelog) - generates a snippet of Markdown that can be added to a CHANGELOG.md.
+
+[conventional-changelog/conventional-changelog](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) - generates a full changelog based on commit history with the option to append to an existing changelog.
```

Comment:
I'd suggest moving these elsewhere for the final changelog.

There's also https://keepachangelog.com (not for generating, but just a high level of why a changelog is a good idea).